### PR TITLE
[Bexley][WW] Implement form for requesting new/replacement containers

### PR DESCRIPTION
**Pull request by nephila-nacrea**
NOTE: Currently on top of https://github.com/mysociety/fixmystreet/pull/5217 .

For https://github.com/mysociety/societyworks/issues/4575.

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm [hunk 1]
```diff
@@ -1236,10 +1237,12 @@ sub construct_bin_request_form {
         for my $container (
             @{ $c->stash->{property}{containers_for_delivery} } )
         {
+            my $id;
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:1240]: Doesn't matter, but can't see why ID was pulled out of the if here, it's never used after.

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm [hunk 2]
```diff
@@ -1273,10 +1274,28 @@ sub construct_bin_request_form {
                     type         => 'Checkbox',
                     label        => $container->{name},
                     option_label => $container->{description},
+                    tags => { toggle => "form-quantity-$id-row" },
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:1275]: Should this only be included if the max is > 1 (I don't think the JS toggle will care, though, so doesn't matter)

File: perllib/FixMyStreet/App/Controller/Waste.pm [hunk 3]
```diff
@@ -859,6 +859,36 @@ sub request : Chained('property') : Args(0) {
             intro => 'request/intro.html',
             check_unique_id => 0,
             next => $next,
+            update_field_list => sub {
```
Comment (dracos) [inline perllib/FixMyStreet/App/Controller/Waste.pm:862]: I see why this is here, and works fine, but I guess ideally we'd want it not to be, given it's Bexley specific :)
Wonder how best to do that. I guess we have the cobrand available in $form->{c} here (I think), so you could move this to a cobrand call_hook `munge_waste_request_page` or something like that, and then call it from here?

**Review by dracos — CHANGES_REQUESTED** [review]
This looks good :)
Couple of tiny comments is all

File: perllib/FixMyStreet/App/Controller/Waste.pm (hunk 3 above)
Comment (nephila-nacrea) [reply perllib/FixMyStreet/App/Controller/Waste.pm:862]: Yeah, this wasn't going to be its permanent place, I need to refactor some things still but wanted to get something working :-) Yeah, I was thinking a call_hook would do the job in some way.

File: perllib/FixMyStreet/Cobrand/Bexley/Waste.pm (hunk 2 above)
Comment (nephila-nacrea) [reply perllib/FixMyStreet/Cobrand/Bexley/Waste.pm:1278]: Yeah, I'm happy to leave this as is.

**Review by dracos — COMMENTED** [review]
Thanks, all looks good :)

**Comment by nephila-nacrea** [comment]
> Thanks, all looks good :)

@dracos, Thanks, just added one more small commit 🙏

**Review by dracos — APPROVED** [review]

**Pull request merged**